exercises/triangle/triangle_test.py

```diff
@@ -1,67 +1,65 @@
 import unittest
 
-from triangle import Triangle, TriangleError
+from triangle import equilateral, isosceles, scalene
```

Comment:
I would prefer these to be `is_equilateral`, `is_isosceles`, and `is_scalene` - it just makes it clearer what these functions do.